### DISCUSSION TITLE
MACE on LAMMPS with dispersion enabled

### DISCUSSION THREAD

**Discussion by asgotz** [ACEsuit/mace#495] [Q&A] [answered]
I am interested in running MD simulations of graphite with the mace potential. Obviously the DFTD3 correction is required for this material, however it seems that DFTD3 is currently not implemented in LAMMPS. Is my only option then running MD simulations using ASE? How much slower should I expect ASE to be compared to LAMMPS? Based on my tests so far I can run 1000 timesteps with 1000 atoms on a GPU in about 21 minutes using LAMMPS and 40 minutes using ASE, which does not seem to be such an enormous difference. However, LAMMPS has a lot more MD features which would make it my first choice if there is a way. I tried modifying the LAMMPS_MACE class to include dispersion from TorchDFTD but it turned out to be quite challenging. What would be the current way to go, and is there anyone (besides me) working to solve this problem?

**Comment by wcwitt** [comment, marked as answer]
If you want D3, I would stick with ASE for now. Those speed differences sound plausible (can vary meaningfully based on model and system size).

We are aware of this issue, but don't have anyone actively working on it yet. I just asked in the LAMMPS Slack if there are any existing efforts to add D3. If not, we should probably address it soon.
